base/docs/helpdb/Base.jl

```diff
@@ -5643,7 +5643,8 @@ julia> deleteat!([6, 5, 4, 3, 2, 1], 1:2:5)
 
 julia> deleteat!([6, 5, 4, 3, 2, 1], (2, 2))
 ERROR: ArgumentError: indices must be unique and sorted
- in deleteat! at array.jl:543
+ in deleteat!(::Array{Int64,1}, ::Tuple{Int64,Int64}) at ./array.jl:571
+ in eval(::Module, ::Any) at ./boot.jl:231...
```

Comment:
I am ok with doing that change. I didn't do that as the original stack traces included `eval(::Module, ::Any)`. 

@tkelman just to confirm: You want me to removing the `eval(::Module, ::Any)` from all (almost all) stack traces in the manual?